.changeset/cuddly-dingos-sell.md

```diff
@@ -0,0 +1,7 @@
+---
+'@skeletonlabs/skeleton-svelte': patch
+'@skeletonlabs/skeleton-react': patch
+---
+
+Show first and last button to be an optional props.
```

Comment:
@phamduylong two quick things with the changeset:

1. They should be prefixed with `chore:`, `bugfix:`, etc same as a the branch. I've missed this on your last couple contributions.
2. The description needs to be a single-line paragraph. As it'll be added as a bullet point list item.

Please refer to any of the generated changelogs to see what I mean.

Reply:
Done that. And no big deal out of my contributions. I'm happy to help.